Common/include/toolboxes/multilayer_perceptron/CIOMap.hpp

```diff
@@ -0,0 +1,87 @@
+/*!
+ * \file CIOMap.hpp
+ * \brief Declaration of input-to-output mapping class for artifical neural networks
+ * \author E. Bunschoten
+ * \version 7.4.0 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2022, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include <cmath>
+#include <string>
+#include <iostream>
+#include <limits>
+#include <cstdlib>
+#include <vector>
+#include "../../CConfig.hpp"
+#include "../../linear_algebra/blas_structure.hpp"
+#include "CLookUp_ANN.hpp"
+
+using namespace std;
+namespace MLPToolbox
+{
+        
+    class CLookUp_ANN;
+
+    class CIOMap
```

Comment:
Please run clang-format on new files and add documentation at least at the class level, i.e. what is the job of this class? Its a map of what into what?